openssl-sys/Cargo.toml

```diff
@@ -25,7 +25,7 @@ bssl-sys = { version = "0.1.0", optional = true }
 [build-dependencies]
 bindgen = { version = "0.64.0", optional = true, features = ["experimental"] }
 cc = "1.0.61"
-openssl-src = { version = "111", optional = true }
+openssl-src = { version = "300", optional = true, features = ["legacy"] }
```

Comment:
Not sure if we want to enable this in the built openssl (or only use it for tests).